agenta-backend/agenta_backend/services/db_manager.py

```diff
@@ -146,7 +183,7 @@ def add_variant_based_on_previous(
 
 def list_app_variants(
     app_name: str = None, show_soft_deleted=False
-) -> List[AppVariant]:
+) -> List[Template]:
```

Comment:
I don't understand this change? Why is list_app_variants is returning templates now?

Reply:
This looks like a typo

Reply:
Yes, that was a typo. I'll fix that now.